app/views/conferences/_details.html.slim

```diff
@@ -6,7 +6,8 @@
     dd = @conference.date_range
     dt Twitter
     dd = format_conference_twitter(@conference.twitter)
-    dt Homepage
-    dd = link_to @conference.url, @conference.url
+    -if @conference.url
```

Comment:
Can you add a `present?` please? When we later edit the conference and leave the url field empty, it will end up with an empty string (which is _truthy_).